cypress/e2e/settings.spec.ts

```diff
@@ -124,6 +133,68 @@ describe("Settings page", () => {
     cy.get("#title").should("have.value", TEST_TITLE)
   })
 
+  it("should be able to update the SEO to a valid value", () => {
+    // Arrange
+    const expected = "www.space.open.gov.sg"
+    cy.contains("label", "SEO").parent().parent().find("input").as("seoInput")
+
+    // Act
+    cy.get("@seoInput").clear().type(expected)
+
+    // Assert
+    cy.saveSettings()
+    cy.get("@seoInput").should("have.value", expected)
+  })
+
+  it("should not be able to input the protocol at the beginning of the url", () => {
+    // Arrange
+    const INVALID_SEO_INPUTS = [
+      `https://${BASE_SEO_LINK}`,
+      `http://${BASE_SEO_LINK}`,
+    ]
+    cy.contains("label", "SEO").parent().parent().find("input").as("seoInput")
+
+    INVALID_SEO_INPUTS.forEach((invalidInput) => {
+      // Act
+      // NOTE: We need to blur for the error message to show
+      // as the validation mode is after the first unfocus
+      cy.get("@seoInput").clear().type(invalidInput).blur()
+
+      // Assert
+      cy.contains("The web domain you have entered is not valid").should(
+        "exist"
+      )
+    })
+  })
+
+  it("should not allow URLs with subdomains for the SEO URL", () => {
+    // Arrange
+    const urlWithSubdomain = "www.open.gov.sg/sub"
```

Comment:
Sorry, to clarify the terminology:
- Domain (or primary domain) is something like `open.gov.sg`
- Subdomain would be something like `space.open.gov.sg`
- Subdirectories would be this case, which is `open.gov.sg/directory`

I think we can rename this to be clear that it is a subdirectory rather than a subdomain (the proper terminology is url-path).

Reply:
sure thing, thanks for catching!